syntax = "proto2";
package spider;

message produce_url_req {
	repeated string urls = 1;
}
